### PR TITLE
Add Scorewatch August Week 1

### PR DESCRIPTION
**Pull request by ThePooN**
### Description

Scorewatch August 1

### Status

WIP, Reviews still welcomed

### Checklist 

- [ ] Banner
- [ ] Scorewatch

### CODE REVIEW

File: news/2017-08-11-scorewatch-august-week-1.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Scorewatch: August Week 1"
+date: 2017-08-11 15:00:00 +0200
+---
+
+Even though several top players have travelled to China this week, things are still taking shape back in the servers! A new 700pp member has joined the club, a special SS set on a difficult osu!mania map, and much more in this edition of Scorewatch!
```
Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: Another player has joined the 700pp club, maaybe?

Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: osu!mania map -> osu!mania beatmap?

File: news/2017-08-11-scorewatch-august-week-1.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+[_RyuK](https://osu.ppy.sh/u/6304246) has broken into the 700pp club! The number 1 Canadian set his mark on the famous map [kradness&Reol - Remote Control](https://osu.ppy.sh/b/774965?m=0) and taking an outstanding 97.76% HDDT 1 miss score to earn an amazing 704pp! Well, I don’t think I need to write much more for this score. Have a look!
```
Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: an amazing 704pp score

File: news/2017-08-11-scorewatch-august-week-1.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+<iframe width="100%" height="315"
+src="https://www.youtube.com/embed/b666KhdtrMU" frameborder="0" allowfullscreen></iframe></br>
+
+An unfamiliar face to newcomers, but definitely a player that has not rusted a bit! [Niko](https://osu.ppy.sh/u/175141) has came back strong with an amazing 98.59% S rank on [Tatsh - IMAGE -MATERIAL- <Version 0>](https://osu.ppy.sh/b/252238), taking the 7th only no mod S on this map, with an accuracy that even beat out [rrtyui](https://osu.ppy.sh/u/352328} himself! What a comeback from this amazing player!
```
Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: has come back

File: news/2017-08-11-scorewatch-august-week-1.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+<iframe width="100%" height="315"
+src="https://www.youtube.com/embed/85Z1LMFMUK8" frameborder="0" allowfullscreen></iframe></br>
+
+Don’t underestimate this play! [ThePooN](https://osu.ppy.sh/u/718454) just set a crazy 98.48% 13x miss score on the hardest difficulty of [And Aoiyuuki - I love Flezlin](https://osu.ppy.sh/s/525461). You may want to catch 2:45 out, as he catches all the jumps in that section like nothing! What a score.
```
Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: has just set

Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: Change dot into exclamation mark on the last sentence?

File: news/2017-08-11-scorewatch-august-week-1.md (hunk 3 above)
Comment (ThePooN) [inline news/2017-08-11-scorewatch-august-week-1.md]: are you sure? i'm pretty sure `came` is right

File: news/2017-08-11-scorewatch-august-week-1.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+Don’t underestimate this play! [ThePooN](https://osu.ppy.sh/u/718454) has just set a crazy 98.48% 13x miss score on the hardest difficulty of [And Aoiyuuki - I love Flezlin](https://osu.ppy.sh/s/525461). You may want to catch 2:45 out, as he catches all the jumps in that section like nothing! What a score!
+
+<iframe width="100%" height="315"
+src="https://player.twitch.tv/?video=v165036056&autoplay=false" frameborder="0" allowfullscreen></iframe></br>
+
+[shinchikuhome](https://osu.ppy.sh/u/3174184) snatched an amazing 97.54% DT FC on the recently ranked [BL-Records - Midway Colors](https://osu.ppy.sh/p/beatmap?b=1251769&m=1) to take away an impressive 656pp score. Yes, this is 270bpm. And yes, this is absolutely insane.
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/0RnuOlD04ug" frameborder="0" allowfullscreen></iframe></br>
+
+[jakads](https://osu.ppy.sh/u/259972) has finally taken the second SS on [Colorful Sounds Port - ETERNAL DRAIN](https://osu.ppy.sh/b/1001780?m=3), taking the number one spot on the map and 646pp in the process! He may have not been the first one to take the 100% accuracy, but his ratio is definitely the best amongst everyone.
```
Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: He may has

File: news/2017-08-11-scorewatch-august-week-1.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/0RnuOlD04ug" frameborder="0" allowfullscreen></iframe></br>
+
+[jakads](https://osu.ppy.sh/u/259972) has finally taken the second SS on [Colorful Sounds Port - ETERNAL DRAIN](https://osu.ppy.sh/b/1001780?m=3), taking the number one spot on the map and 646pp in the process! He may have not been the first one to take the 100% accuracy, but his ratio is definitely the best amongst everyone. 
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/QJ6GyjY5qrY" frameborder="0" allowfullscreen></iframe></br>
+
+NOTABLE MENTIONS
+-----
+
+[WubWoofWolf](https://osu.ppy.sh/u/39828) has shocked the osu! community with an amazing play on [Halozy - Genryuu Kaiko](https://osu.ppy.sh/b/433005)! Slapping on the HD mod, he went on to take an incredible 99.15% HD 4 miss score to land 4th on the leaderboards and even earn 408pp in the process! Wolf at its finest.
```
Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: HD -> Hidden

File: news/2017-08-11-scorewatch-august-week-1.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+
+SCORESHOW WITH ZSEIKIMATSU
```
Comment (TPGPL) [inline news/2017-08-11-scorewatch-august-week-1.md]: Is this header even correct?

Comment (ThePooN) [inline news/2017-08-11-scorewatch-august-week-1.md]: yes

File: news/2017-08-11-scorewatch-august-week-1.md (hunk 6 above)
Comment (ThePooN) [inline news/2017-08-11-scorewatch-august-week-1.md]: we've been using short mods names for ages

**Comment by Raveille** [comment]
Will wait for scoreshow.

**Comment by Ephemeralis** [comment]
This good to go, or still waiting on scoreshow?

**Comment by ThePooN** [comment]
tumblr in 2017
😅

**Comment by Ephemeralis** [comment]
No need for tumblr anymore, will get to the banner later on today.

**Comment by Ephemeralis** [comment]
I reworded the entry paragraph slightly and refreshed dates + filenames. Give a quick check and holler at me if you notice anything wrong, and we'll get this out today.

File: news/2017-08-17-scorewatch-august-weeks-1-and-2.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/b666KhdtrMU" frameborder="0" allowfullscreen></iframe></br>
+
+An unfamiliar face to newcomers, but definitely a player that has not rusted a bit! [Niko](https://osu.ppy.sh/u/175141) has come back strong with an amazing 98.59% S rank on [Tatsh - IMAGE -MATERIAL- <Version 0>](https://osu.ppy.sh/b/252238), taking the 7th only no mod S on this map, with an accuracy that even beat out [rrtyui](https://osu.ppy.sh/u/352328) himself! What a comeback from this amazing player!
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/85Z1LMFMUK8" frameborder="0" allowfullscreen></iframe></br>
+
+[HappyStick](https://osu.ppy.sh/u/256802) has produced an incredible score on [Tech N9ne - Worldwide Choppers](https://osu.ppy.sh/b/345189?m=0), in fact, getting the first ever no mod S on the map and the No. 1 spot with a 98.79% 2225x combo play to earn 372pp. What a shame that an FC could have happened, but nonetheless, enjoy this livestream moment!
```
Comment (this1neguy) [inline news/2017-08-17-scorewatch-august-weeks-1-and-2.md]: would remove "in fact," and change last line to something like "while a shame the fc didn't happen, enjoy this livestream moment nonetheless"

**Pull request merged**